src/GitProjectHealth-Model-Importer/GitModelImporter.class.st

```diff
@@ -49,6 +49,18 @@ GitModelImporter >> blockEqualityOn: aSymbol [
 	  (existing perform: aSymbol) = (new perform: aSymbol) ]
 ]
 
+{ #category : #'as yet unclassified' }
+GitModelImporter >> blockEqualityOn: aSymbol andOn: aSecondSymbol [
+	^ (self blockEqualityOn: aSymbol) and: [ self blockEqualityOn: aSecondSymbol ]
+]
+
+{ #category : #equality }
+GitModelImporter >> blockForBranchEquality [
+	^ [ :existing :new |
+	  (existing name = new name) and: [ existing repository project id = new repository project id ] ]
```

Comment:
Maybe we could have use https://modularmoose.org/developers/create-new-metamodel/#generating-custom-equality-check of the generator to ease the `blockFor`methods
To study in the future